{{cookiecutter.project_slug}}/requirements/base.txt

```diff
@@ -29,7 +29,7 @@ uvicorn[standard]==0.13.3  # https://github.com/encode/uvicorn
 
 # Django
 # ------------------------------------------------------------------------------
-django==3.0.11  # pyup: < 3.1  # https://www.djangoproject.com/
+django==3.1.6  # pyup: < 3.1  # https://www.djangoproject.com/
```

Comment:
We also need to update the PyUP filter to keep receiving updates:

```suggestion
django==3.1.6  # pyup: < 3.2  # https://www.djangoproject.com/
```

Reply:
Well spotted!